src/Anchu/Ftp/Ftp.php

```diff
@@ -30,11 +30,15 @@ public function connect($config)
     {
         if(!isset($config['port']))
             $config['port'] = 21;
-
-        $connectionId = ftp_connect($config['host'],$config['port']);
-        $loginResponse = ftp_login($connectionId, $config['username'], $config['password']);
-        ftp_pasv($connectionId, $config['passive']);
-        ftp_set_option($connectionId, FTP_TIMEOUT_SEC, 300);
+        if(!isset($config['timeout']))
+            $config['timeout'] = 30;
```

Comment:
The default php-ftp timeout value is 90 seconds. Hence I think timeout value should be 90 if user didn't supply any value. So can you update default timeout value to 90?